ansible/aee/ansibleee-external-inventory.yaml

```
@@ -0,0 +1,44 @@
+apiVersion: redhat.com/v1alpha1
```

```suggestion
apiVersion: ansibleee.openstack.org/v1alpha1
```
we had to change it
https://github.com/openstack-k8s-operators/openstack-operator/pull/136#discussion_r1071463960

---

yes we did i have not updated it since you changed it.
its on my todo list but I'm intentionally not working on this this week
ill update it next time i work on this.

also, we need to modify the ansibleEE operator to have a status filed which will monitor the status of the job and also clean up the pod when the job completes correctly.

as it stand right now we cant tell if the playbook completed correctly form the operator point of view.